lib/core/versions/latest/TransactionProcessor.ts

```diff
@@ -88,6 +90,18 @@ export default class TransactionProcessor implements ITransactionProcessor {
         `Operation count ${operationCountInAnchorFile} in anchor file exceeded limit of : ${paidOperationCount}`);
     }
 
+    // Verify required lock if one was needed.
+    const valueTimeLock = anchorFile.model.writerLock
+                          ? await this.blockchain.getValueTimeLock(anchorFile.model.writerLock)
+                          : undefined;
+
+    ValueTimeLockVerifier.verifyLockAmountAndThrowOnError(
+      valueTimeLock,
+      operationCountInAnchorFile,
```

Comment:
Sanity: should this be `paidOperationCount`?

Reply:
Good catch. Done.